lenskit-gradle/src/main/groovy/org/lenskit/gradle/LenskitPlugin.groovy

```diff
@@ -51,6 +53,13 @@ public class LenskitPlugin implements Plugin<Project> {
                 if (prop.type != String) {
                     val = prop.type.metaClass.invokeConstructor(val)
                 }
+
+                // if the type is list update the val using strtokenizer
+                if (prop.type == List) {
```

Comment:
This should be integrated with the previous `if`, probably before it, so we have:

```java
if (prop.type == List) {
    // setup code
} else if (prop.type != String) {
    // original code
}
```